app/main.py

```diff
@@ -1,3 +1,112 @@
+        if isinstance(other, Distance):
+            raise TypeError("'__mul__' method should "
+                            "not accept Distance instance")
+        elif isinstance(other, (int, float)):
+            return Distance(km=self.km * other)
+        else:
+            raise TypeError(
+                f"Unsupported operand type(s) for *: "
+                f"'Distance' and '{type(other).__name__}'"
+            )
+
+    def __truediv__(self, other: Any) -> "Distance":
+        if isinstance(other, Distance):
+            raise TypeError("'__truediv__' method should "
+                            "not accept Distance instance")
+        elif isinstance(other, (int, float)):
+            if other == 0:
+                raise ZeroDivisionError("Cannot divide by zero")
+            num = round(self.km / other, 2)
+        else:
+            raise TypeError(
+                f"Unsupported operand type(s) for /: "
+                f"'Distance' and '{type(other).__name__}'"
+            )
+        return Distance(num)
+
+    def __lt__(self, other: Any) -> bool:
+        if isinstance(other, Distance):
+            return self.km < other.km
+        elif isinstance(other, (int, float)):
+            return self.km < other
+        else:
+            raise TypeError(f"Unsupported operand type(s) for <: "
+                            f"'Distance' and '{type(other).__name__}'"
+                            )
+
+    def __gt__(self, other: Any) -> bool:
+        if isinstance(other, Distance):
+            return self.km > other.km
+        elif isinstance(other, (int, float)):
+            return self.km > other
+        else:
+            raise TypeError(f"Unsupported operand type(s) for >: "
+                            f"'Distance' and '{type(other).__name__}'"
+                            )
+
+    def __eq__(self, other: Any) -> bool:
+        if isinstance(other, Distance):
+            return self.km == other.km
+        elif isinstance(other, (int, float)):
+            return self.km == other
+        else:
+            raise TypeError(f"Unsupported operand type(s) for ==: "
+                            f"'Distance' and '{type(other).__name__}'"
+                            )
+
+    def __le__(self, other: Any) -> bool:
+        if isinstance(other, Distance):
+            return self.km <= other.km
+        elif isinstance(other, (int, float)):
+            return self.km <= other
+        else:
+            raise TypeError(f"Unsupported operand type(s) for <=: "
+                            f"'Distance' and '{type(other).__name__}'"
+                            )
+
+    def __ge__(self, other: Any) -> bool:
+        if isinstance(other, Distance):
+            return self.km >= other.km
+        elif isinstance(other, (int, float)):
+            return self.km >= other
+        else:
+            raise TypeError(f"Unsupported operand type(s) for >=: "
+                            f"'Distance' and '{type(other).__name__}'"
```

Comment:
Ensure that the error message accurately reflects the '>=' operator for the `__ge__` method.